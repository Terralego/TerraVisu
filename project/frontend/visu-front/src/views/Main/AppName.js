import React from 'react';

export const AppName = ({ title = 'TerraVisu', version = 'v1.0' }) => (
  <div className="bp3-dark appName">
    <span className="appName-title">
      { title }
    </span>
    <span className="appName-version">{ version }</span>
  </div>
);


export default AppName;
