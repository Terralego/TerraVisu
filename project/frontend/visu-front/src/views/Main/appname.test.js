import React from 'react';
import renderer from 'react-test-renderer';

import AppName from './AppName';

it('should render', () => {
  const tree = renderer.create(
    <AppName />,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
