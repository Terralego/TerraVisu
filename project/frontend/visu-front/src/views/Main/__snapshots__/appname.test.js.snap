// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<div
  className="bp3-dark appName"
>
  <span
    className="appName-title"
  >
    TerraVisu
  </span>
  <span
    className="appName-version"
  >
    v1.0
  </span>
</div>
`;
